corporate/tests/test_stripe.py

```diff
@@ -218,8 +218,8 @@ def setUp(self, *mocks: Mock) -> None:
         realm = get_realm('zulip')
         seat_count = get_seat_count(realm)
         assert(seat_count >= 6)
-        for user in UserProfile.objects.filter(realm=realm, is_active=True, is_bot=False, is_guest=False) \
-                                       .exclude(email__in=[
+        for user in UserProfile.objects.filter(realm=realm, is_active=True, is_bot=False) \
+                                       .exclude(role=UserProfile.GUEST, email__in=[
```

Comment:
I think this change is wrong; `.exclude(cond_a, cond_b)`  excludes things that match both `cond_a` and `cond_b`, not things that match either of them.